ethereum/contracts/BasicInboundChannel.sol

```diff
@@ -56,11 +56,14 @@ contract BasicInboundChannel {
     }
 
     function processMessages(Message[] calldata _messages) internal {
+        // Using tempNonce variable to store nonce value
```

Comment:
This comment doesn't really explain anything. Instead, maybe explain that we are caching the nonce in a memory variable for gas optimization, or something like that.

Reply:
Yeh, in general comments should explain "Why" something is done, not "what" is being done. Variable names should explain "what" is being done.